tests/functions/test_deconcatenate_column_preserve_position.py

```diff
@@ -0,0 +1,29 @@
+import pytest
```

Comment:
I’m so sorry, I should have made this clearer in the docs. This test should be in `test_deconcatenate_column.py`, because all tests for a particular function go in a single `.py` file. I don’t want to hold up this PR, so I’ll let itgo, and PR back the correct changes into `dev`, so don’t worry about it :smile:.

Reply:
Oops, sorry about that! (now I know what to do if similar situations are to happen in future). Thanks a lot! 
Also, the entire CI process feels like magic~ I'll check if in the scipy talk you have talked about how this was set up, and how the stacks behind it was chosen.